src/RhythmRunner.cpp

```diff
@@ -42,9 +42,14 @@
 #define GLEE_OVERWRITE_GL_FUNCTIONS
 #include "glee.hpp"
 
+#include "game_state/GameState.h"
+
 using namespace std;
```

Comment:
delete this don't use namespaces according to google style
![1378960_630x354](https://cloud.githubusercontent.com/assets/8975458/22315506/f6a2877a-e31b-11e6-99fd-3a139b1c9017.jpeg)